Dockerfile.ubuntu

```diff
@@ -60,6 +49,3 @@ RUN pip install -r /opt/file-metadata/test-requirements.txt
 
 # Install package
 RUN pip install /opt/file-metadata --upgrade
-
```

Comment:
We need some command to run in the end to test the successful install. At least an import test.